_data/vnext_faq.yml

```diff
@@ -41,7 +41,7 @@ extenders:
            The bypass scheme remains and you still can claim conformance for services that provides only public endpoints that doesn't require authentication. 
 
        - number: 6
-         question: Does the dependency changes mean that I need Java 17 to run Zowe
+         question: Do the dependency changes mean that I need Java 17 to run Zowe
```

Comment:
This should end w/ a question mark.
Also, b/c it's not great to supply a one-word answer, maybe consider something like: Do the dependency changes **mean/result in/pick one** new requirements to run Zowe? Answer: Yes. The new dependencies require Java 17 to run Zowe.